src/main/webapp/app/exercises/quiz/shared/questions/short-answer-question/short-answer-question.component.html

```diff
@@ -51,8 +51,8 @@ <h4 class="question-title-display">
                             class="short-answer-question-container__input"
                             style="background: grey"
                             disabled
-                            value="{{ getSubmittedTextForSpot(element) !== undefined ? getSubmittedTextForSpot(element).text : '' }}"
-                            size="{{ getSubmittedTextForSpot(element) !== undefined ? getSubmittedTextForSpot(element).text!.length + 2 : 5 }}"
+                            value="{{ getSubmittedTextForSpot(element)?.text !== undefined ? getSubmittedTextForSpot(element).text : '' }}"
```

Comment:
I think you can simplify that line with `getSubmittedTextForSpot(element)?.text ?? ''`

Reply:
On another though, wouldn't it be better to store `getSubmittedTextForSpot(element)?.text` as a variable so that we don't have to do method calls / make the html longer?

Reply:
@ivanchimeno can you provide an example of how to use the operator `??` in this case? I tried this and it throws a compilation error `Expression expected`

Reply:
@ivanchimeno I agree with you it would be better to store the appropriate values as variables in the component but this would require a refactor to the whole component as it uses quite a lot of getters including `getSubmittedTextForSpot` `getSampleSolutionForSpot`, `getBackgroundColourForInputField`, `isSubmittedTextCompletelyCorrect`. I think this is out of scope for this bugfix, where we just want to fix the bugs found